habitat-lab/habitat/sims/habitat_simulator/habitat_simulator.py

```diff
@@ -351,6 +351,12 @@ def create_sim_config(
             },
         )
 
+        # configure default navmesh parameters to match the configured agent
+        sim_config.navmesh_settings = habitat_sim.nav.NavMeshSettings()
+        sim_config.navmesh_settings.set_defaults()
+        sim_config.navmesh_settings.agent_radius = agent_config.radius
```

Comment:
@jimmytyyang should know how this is done in more detail.
I believe we use a radius of 0.3 as indicated by [this config](https://github.com/facebookresearch/habitat-lab/blob/db1539ac4cb11a1c50cf6bba498f929e8d73daa0/habitat-lab/habitat/config/benchmark/rearrange/play_spot.yaml#LL47C1-L47C1). Not sure if that is pushed down into agent_radius field or needs additional config to be reflected here.

In general I think this particular pathways can now use some design now that we have exposed the full range of navmesh parameters to programmatic configuration from lab.

Reply:
Thanks both! Right Now I am using a radius of 0.25 so that it can navigate in the tighter space and it is consistent with the real robot dimension. This radius is configured in the yaml file.